demo/Signs.swift

```diff
@@ -1067,6 +1175,767 @@ extension Sign {
                 return Asset.Signs.warningRoadworksCN
             case .warningSecondRoadLeft:
                 return Asset.Signs.warningSecondRoadLeftCN
+            case .regulatoryNoTurnOnRedText:
```

Comment:
Why are there so many new nil cases for china? Can't we just use `default`

Reply:
The reason why I left cases explicitly is the "type-safety". It allows us to control adding of new types explicitly, I mean the compiler will produce error. In case when we have `default` there's no error, IMHO it is more error-prone. Does that make sense?

In general I don't like the way it's written right now. In a separate issue I proposed to add more dynamic behavior, it will allow us to reduce such code (and reduce `Sign.swift` heavily).

Reply:
Oh, I've just created the same issue)
And no, I think it won't be handy and moreover a big part of these signs just aren't related to the countries with the current classifiers

Reply:
:)

I agree, currently it depends on a common `SignType`. It has a flat structure and we can't guarantee that specific sign will be recognized by specific classifiers only.

At the same time we decided to leave there (I mean in Core) a flat structure in order to allow other developers to work with specific types of signs only. E.g. someone wants to track only "speed limits" in different countries. According to @deniskoronchik we're not going to change the behavior for now.

Anyway it must be improved.

Reply:
Update: squash nil-cases under common `default`.